src/plugins/main.js

```diff
@@ -88,6 +88,12 @@ export default async ({ app, route, store, req }) => {
   // Lazy-load translations
   if (lazy) {
     const { loadLanguageAsync } = require('./utils')
+
+    // Load fallback lang
+    if(app.i18n.fallbackLocale) {
+      loadLanguageAsync(app.i18n, app.i18n.fallbackLocale)
```

Comment:
- Need to await
- No need to call it if fallback is the same as current
- I think it's not the best place to do it as it will trigger for every request. It should be done at init stage instead.

Reply:
I have fix that but i don't know where i can move that. I think in `index.js` but this need change `loadLanguageAsync`

Reply:
I think it actually has to be done on each request so disregard my last point.

This fix looks good to me (apart from missing space after `if` ;)).

Reply:
Its done